team.html

```diff
@@ -191,403 +191,455 @@ <h2 id="contributors">Contributors<a class="paralink" href="#contributors" title
+			<li>Frédéric Chapoton</li>
+			<li>Frédéric Grollier</li>
+			<li>Gabriel Brammer</li>
+			<li>Gabriel Perren</li>
+			<li>Geert Barentsen</li>
+			<li>George Galvin</li>
+			<li>Georgiana Ogrean</li>
+			<li>Gerrit Schellenberger</li>
+			<li>Giang Nguyen</li>
+			<li>Giorgio Calderone</li>
+			<li>Graham Kanarek</li>
+			<li>Grant Jenks</li>
+			<li>Gregory Dubois-Felsmann</li>
+			<li>Gregory Simonian</li>
+			<li>Griffin Hosseinzadeh</li>
+			<li>Gustavo Bragança</li>
+			<li>Gyanendra Shukla</li>
+			<li>Hannes Breytenbach</li>
+			<li>Hans Moritz Günther</li>
+			<li>Harry Ferguson</li>
+			<li>Helen Sherwood-Taylor</li>
+			<li>Himanshu Pathak</li>
+			<li>homeboy445</li>
+			<li>Hugo Buddelmeijer</li>
+			<li>Humna Awan</li>
+			<li>iamsoto</li>
+			<li>ikkamens</li>
+			<li>Inada Naoki</li>
+			<li>J . Goutin</li>
+			<li>J . Xavier Prochaska</li>
+			<li>Jake VanderPlas</li>
+			<li>Jakob Maljaars</li>
+			<li>James Davies</li>
+			<li>James Dearman</li>
+			<li>James Noss</li>
+			<li>James Taylor</li>
+			<li>James Tocknell</li>
+			<li>James Turner</li>
+			<li>Jan Skowron</li>
+			<li>Jane Rigby</li>
+			<li>Jani Šumak</li>
+			<li>Jason Segnini</li>
+			<li>Javier Pascual Granado</li>
+			<li>JC Hsu</li>
+			<li>Jean Connelly</li>
+			<li>Jeff Taylor</li>
+			<li>Jeffrey McBeth</li>
+			<li>Jero Bado</li>
+			<li>Jerry Ma</li>
+			<li>jimboH</li>
+			<li>Joanna Power</li>
+			<li>Joe Hunkeler</li>
+			<li>Joe Lyman</li>
+			<li>Joe Philip Ninan</li>
+			<li>John Fisher</li>
+			<li>John Parejko</li>
+			<li>Johnny Greco</li>
+			<li>Jonas Große Sundrup</li>
+			<li>Jonathan Eisenhamer</li>
+			<li>Jonathan Foster</li>
+			<li>Jonathan Sick</li>
+			<li>Jonathan Whitmore</li>
+			<li>Jörg Dietrich</li>
+			<li>Jose Sabater</li>
+			<li>Joseph Jon Booker</li>
+			<li>Joseph Long</li>
+			<li>Joseph Ryan</li>
+			<li>Joseph Schlitz</li>
+			<li>José Sabater Montes</li>
+			<li>Juan Luis Cano Rodríguez</li>
+			<li>Juanjo Bazán</li>
+			<li>Julien Woillez</li>
+			<li>Jurien Huisman</li>
+			<li>Kacper Kowalik</li>
+			<li>Karan Grover</li>
+			<li>Karl Gordon</li>
+			<li>Karl Vyhmeister</li>
+			<li>Karl Wessel</li>
+			<li>Katrin Leinweber</li>
+			<li>Kelle Cruz</li>
+			<li>Kevin Gullikson</li>
+			<li>Kevin Sooley</li>
+			<li>Kewei Li</li>
+			<li>Kieran Leschinski</li>
+			<li>Kirill Tchernyshyov</li>
+			<li>Kris Stern</li>
+			<li>Kristin Berry</li>
+			<li>Kyle Barbary</li>
+			<li>Kyle Oman</li>
+			<li>Larry Bradley</li>
+			<li>Laura Hayes</li>
+			<li>Laura Watkins</li>
+			<li>Lauren Glattly</li>
+			<li>Laurie Stephey</li>
+			<li>Leah Fulmer</li>
+			<li>Lee Spitler</li>
+			<li>Lehman Garrison</li>
+			<li>Lennard Kiehl</li>
+			<li>Leo Singer</li>
+			<li>Leonardo Ferreira</li>
+			<li>Lia Corrales</li>
+			<li>Lingyi Hu</li>
+			<li>Lisa Martin</li>
+			<li>Lisa Walter</li>
+			<li>Ludwig Schwardt</li>
+			<li>Luigi Paioro</li>
+			<li>Luke G. Bouma</li>
+			<li>Luke Kelley</li>
+			<li>luz paz</li>
```

Comment:
Isn't this an actual proper name?
```suggestion
			<li>Luz Paz</li>
```
(I know it's derived from the core repo name list, so I can correct it there if you confirm I'm not missing something, @astrofrog )

Reply:
Sounds good, feel free to add to mailmap